misc/docs-linter/main_test.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"context"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestEmptyPathError(t *testing.T) {
+	cfg := &cfg{
+		docsPath: "",
+	}
+
+	ctx, cancelFn := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancelFn()
+
+	err := execLint(cfg, ctx)
+	assert.ErrorIs(t, err, ErrEmptyPath)
```

Comment:
This can be inlined

Reply:
Inlined the errors in the tests: [a38e987](https://github.com/gnolang/gno/pull/2125/commits/a38e98744121ef0c38a6d47f6156f75806aba45b)